packages/loot-core/src/server/transactions/import/parse-file.test.ts

```
@@ -103,11 +109,42 @@
   }, 45000);
 
+  test('import notes are respected when importing', async () => {
+    prefs.loadPrefs();
+    await db.insertAccount({ id: 'one', name: 'one' });
+
+    // Test with importNotes enabled
+    const { errors: errorsWithNotes } = await importFileWithRealTime(
+      'one',
+      __dirname + '/../../../mocks/files/data.ofx',
+      null,
+      { importNotes: true }
```

ESLint

[error] Insert `,`

(prettier/prettier)